app/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentDownloadController.java

```diff
@@ -260,6 +256,19 @@ void microarrayExperimentDownload(String experimentAccession,
                                         experiment.getAccession(), ExperimentFileType.SUMMARY_PDF));
                         break;
 
+                    case PROTEOMICS_DIFFERENTIAL:
+                        paths.add(experimentFileLocationService.getFilePath(
+                                experiment.getAccession(), ExperimentFileType.CONDENSE_SDRF))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.CONFIGURATION))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.PROTEOMICS_RAW_QUANT))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.IDF))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.SUMMARY_PDF));
```

Comment:
What happens if some of these files are not present? does it fail? Some of these will be optional (like the summary PDF).

Reply:
It will not fail instead it shows an alert to the user that this file is not available and if they want to continue. When user clicks continue then it will download all available files.